config.md

```diff
@@ -349,6 +349,8 @@ For Windows based systems the user structure has the following fields:
     This MUST be set if the target platform of this spec is `windows`.
 * **`solaris`** (object, OPTIONAL) [Solaris-specific configuration](config-solaris.md).
     This MAY be set if the target platform of this spec is `solaris`.
+* **`vm`** (object, OPTIONAL) [Virtual-machine-specific configuration](config-vm.md).
```

Comment:
You'll also want to add virtual-machine entries [here][1] and [here][2].

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.1/spec.md#platforms
[2]: https://github.com/opencontainers/runtime-spec/blob/v1.0.1/spec.md#table-of-contents

Reply:
done now